modules/examiner/php/NDB_Form_examiner.class.inc

```diff
@@ -54,16 +54,23 @@ class NDB_Form_Examiner extends NDB_Form
         }
 
         if ($useCertification) {
-            $centerID = $DB->pselectOne(
-                "SELECT centerID 
-                 FROM examiners
-                 WHERE examinerID=:EID",
+            $cids      = $DB->pselect(
+                "SELECT epr.centerID 
+                 FROM examiners e 
+                  LEFT JOIN examiners_psc_rel epr ON (e.examinerID=epr.examinerID)
+                 WHERE e.examinerID=:EID",
                 array('EID' => $this->identifier)
             );
+            $centerIDs =array();
+            foreach ($cids as $k=>$row) {
+                $centerIDs[] = $row['centerID'];
+            }
 
+            // Access is only granted if user ENGLOBES all sites of examiner
```

Comment:
Why is "ENGLOBES" all caps? And is there a more.. well known.. word that can be used instead so that people don't need a thesaurus to understand this comment?